docs/src/pages/css-in-js/api/api.md

```diff
@@ -111,63 +111,49 @@ export default function MyComponent() {
 
 ## `ServerStyleSheets`
 
-This is a class helper to handle server-side rendering.
-The instance offers the following API.
-
-```js
-import { ServerStyleSheets } from '@material-ui/styles';
-
-const sheets = new ServerStyleSheets();
-```
-
-### `sheets.collect(node) => void`
-
-The method wraps your node in a provider element.
+This is a class helper to handle server-side rendering. [You can follow our guide for a practical approach](/guides/server-rendering/).
 
 ```jsx
 import ReactDOMServer from 'react-dom/server';
+import { ServerStyleSheets } from '@material-ui/styles';
 
+const sheets = new ServerStyleSheets();
 const html = ReactDOMServer.renderToString(sheets.collect(<App />));
-```
-
-### `sheets.toString() => CSS string`
-
-The method returns the collected styles. It's a CSS string.
-
-```js
-const css = sheets.toString();
+const cssString = sheets.toString();
 
 const response = `
 <!DOCTYPE html>
 <html>
   <head>
-    <style id="jss-server-side">${css}</style>
+    <style id="jss-server-side">${cssString}</style>
   </head>
-  <body>
-    ...
-  </body>
+  <body>${html}</body>
 </html>
 `;
 ```
 
-### `sheets.getStyleElement() => CSS React node`
+### `new ServerStyleSheets([options])`
 
-The method is an alternative to `toString()` when you are rendering the whole page with React.
+The instantiation accepts an options object as a first argument.
 
-```jsx
-const style = sheets.getStyleElement();
-
-const response = (
-  <html lang="en" dir="ltr">
-    <Head>
-      {style}
-    </Head>
-    <body>
-      <Main />
-    </body>
-  </html>
-);
-```
+1. `options` (*Object* [optional]): The options are spread as props to the [`StylesProvider`](#stylesprovider) component.
+
+### `sheets.collect(node) => React element`
+
+The method wraps your React node in a provider element.
+It collects the style sheets during the rendering so they can be later sent to the client.
+
+### `sheets.toString() => CSS string`
+
+The method returns the collected styles.
+
+⚠️ You should call `.collect()` before using this method.
+
+### `sheets.getStyleElement() => CSS React element`
+
+The method is an alternative to `.toString()` when you are rendering the whole page with React.
+
+⚠️ You should call `.collect()` before using this method.
```

Comment:
```suggestion
⚠️ You must call `.collect()` before using this method.
```

Could you explain a use case for calling `toString` before `collect`?

Reply:
There is no use case for calling toString before collect.

Reply:
But your API allows this and you word it so that it's valid.

Reply:
I have reproduced styled-components API. I haven't tried to be smart about it.

Reply:
> But your API allows this and you word it so that it's valid.

Where is the wording allowing it?

Reply:
It would've helped if you either mentioned this or included this in the docs.

Reply:
The only reference to the reproduction of the styled components API is in the [March Update](https://material-ui.com/blog/march-2019-update/). Are all your concerns addressed?